cg/meta/workflow/fluffy.py

```diff
@@ -267,23 +227,25 @@ def make_sample_sheet(self, case_id: str, dry_run: bool) -> None:
         """
         Create SampleSheet.csv file in working directory and add desired values to the file
         """
-        latest_flow_cell: Flowcell = self.status_db.get_latest_flow_cell_on_case(family_id=case_id)
-        sample_sheet_housekeeper_path = self.get_sample_sheet_housekeeper_path(
-            flowcell_name=latest_flow_cell.name
-        )
-        sample_sheet_workdir_path = Path(self.get_sample_sheet_path(case_id=case_id))
-        LOG.info(
-            "Writing modified csv from %s to %s",
-            sample_sheet_housekeeper_path,
-            sample_sheet_workdir_path,
+        flow_cell: Flowcell = self.status_db.get_latest_flow_cell_on_case(case_id)
+        sample_sheet_housekeeper_path: Path = self.get_sample_sheet_housekeeper_path(flow_cell.name)
+        flow_cell_sample_type = get_sample_type_from_sequencer_type(flow_cell.sequencer_type)
+
+        sample_sheet: SampleSheet = get_sample_sheet_from_file(
+            infile=sample_sheet_housekeeper_path,
+            flow_cell_sample_type=flow_cell_sample_type,
```

Comment:
Maybe the `flow_cell_sample_type` can be made into an optional argument so we can hide away the logic for determining the sample type in the `get_sample_sheet_from_file` instead?

Reply:
What I see is that is needed for validating the sample sheet. What I am thinking is that maybe you need specific functions for the Fluffy sample sheet and not use the ones from the normal sample sheets

Reply:
Lets do it in a separate PR

Reply:
Ah, missed your comment. The fluffy "sample sheet" (more like a pipe line config) uses data from the existing sample sheet for some reason. If we had refined this issue further and also increased the scope, we probably would have ended up removing this completely